### PR TITLE
Add documentation for Multiple ACS URL API support for custom SAML apps.

### PR DESCRIPTION
**Pull request by shantanusardal-okta**
## Description:
- **What's changed?** Add documentation for Multiple ACS URL API support for custom SAML apps.
- **Is this PR related to a Monolith release?** 2020.07.0

### Resolves:

* [OKTA-304517](https://oktainc.atlassian.net/browse/OKTA-304517)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 1]
```diff
@@ -6198,3 +6232,27 @@ The application CSR object defines a certificate signing request for a signature
 | csr              | Base64 encoded CSR in DER format                             | String                                                                      | TRUE     | TRUE   | TRUE     |           |           |            |
 | id               | unique identifier for the CSR                                | String                                                                      | FALSE    | TRUE   | TRUE     |           |           |            |
 | kty              | cryptographic algorithm family for the CSR's keypair         | String                                                                      | FALSE    | FALSE  | TRUE     |           |           |            |
+
+### ACS Endpoint Object
+
+The ACS endpoint containing the ACS URI and index of the URI.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:6238]: The ACS endpoint that contains the ACS URI and the index of the URI.

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 2]
```diff
@@ -6198,3 +6232,27 @@ The application CSR object defines a certificate signing request for a signature
...
+#### ACS Endpoint properties
+
+| Property         | Description                                                  | DataType                                                                    | Nullable | Unique | Readonly | MinLength | MaxLength | Validation |
+| ---------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------- | -------- | ------ | -------- | --------- | --------- | ---------- |
+| url              | URL of the ACS                                               | String                                                                      | FALSE    | FALSE  | FALSE    |           | 1024      | [URL](http://tools.ietf.org/html/rfc3986)           |
+| index            | index of the URL in the array of ACS endpoints               | Number                                                                      | FALSE    | TRUE   | FALSE    | 0         |           |            |
+
+Property details
+
+ * `url` cannot have query or fragment parameters.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:6257]: we try to keep a "conversational tone" - so we use contractions:

* 'url' can't have query or fragment parameters.

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 3]
```diff
@@ -6198,3 +6232,27 @@ The application CSR object defines a certificate signing request for a signature
...
+ * `index` has to be a non-negative number, and cannot be duplicated in a set of ACS endpoints configured for an app.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:6258]: no need for the comma before and

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 4]
```diff
@@ -842,6 +842,8 @@ Adds a SAML 2.0 application. This application is only available to the org that
 
 | Parameter             | Description                                     | DataType                             | Nullable | Unique | Validation                              |
 | --------------------- | ----------------------------------------------- | ------------------------------------ | -------- | -----  | --------------------------------------  |
+| allowMultipleAcsEndpoints       | Determines whether the app allows configuring multiple ACS URIs                                         | Boolean                                              | FALSE    | FALSE  |                                           |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:845]: Determines whether the app allows you to configure multiple ACS URIs

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 5]
```diff
@@ -842,6 +842,8 @@ Adds a SAML 2.0 application. This application is only available to the org that
...
+| acsEndpoints          | A maximum of 100 ACS endpoints                                                                                    | Array of [ACS Endpoints](#acs-endpoint-object)       | TRUE     | FALSE  |                                           |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:846]: An array of [ACS endpoints](#acs-endpoint-object). You can configure a maximum of 100 URIs.

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 1 above)
Comment (shantanusardal-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: done

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 2 above)
Comment (shantanusardal-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: done

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 3 above)
Comment (shantanusardal-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: done

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 4 above)
Comment (shantanusardal-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: done

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md (hunk 5 above)
Comment (shantanusardal-okta) [reply packages/@okta/vuepress-site/docs/reference/api/apps/index.md]: done

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 6]
```diff
@@ -842,6 +842,8 @@ Adds a SAML 2.0 application. This application is only available to the org that
 
 | Parameter             | Description                                     | DataType                             | Nullable | Unique | Validation                              |
 | --------------------- | ----------------------------------------------- | ------------------------------------ | -------- | -----  | --------------------------------------  |
+| allowMultipleAcsEndpoints       | Determines whether the app allows you to configure multiple ACS URIs                                    | Boolean                                              | FALSE    | FALSE  |                                           |
+| acsEndpoints          | An array of ACS endpoints. You can configure a maximum of 100 endpoints.                                          | Array of [ACS Endpoints](#acs-endpoint-object)       | TRUE     | FALSE  |                                           |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:846]: Remove "of [ACS Endpoints](#acs-endpoint-object) " at the end and do the link like below:

An array of [ACS Endpoints](#acs-endpoint-object). You can configure a maximum of 100 endpoints.

**Pull request merged**